lib/glueby/internal/wallet.rb

```diff
@@ -84,8 +84,8 @@ def balance(only_finalized = true)
 
       # @param only_finalized [Boolean] The flag to get a UTXO with status only finalized
       # @param label [String] This label is used to filtered the UTXOs with labeled if a key or Utxo is labeled.
-      #                    - If label is not specified (label=nil), all UTXOs will be returned.
-      #                    - If label=:unlabeled, only unlabeled UTXOs will be returned.
+      #                    - If label is not specified or label=:unlabeled, only unlabeled UTXOs will be returned.
+      #                    - If label=:all, all UTXOs will be returned.
       def list_unspent(only_finalized = true, label = nil)
         wallet_adapter.list_unspent(id, only_finalized, label)
       end
```

Comment:
The label default value should be `:unlabeled`.
And also the label variable should be set :unlabeled if the label argument is nil.

```suggestion
      #                    - If label is nil or :unlabeled, only unlabeled UTXOs will be returned.
      #                    - If label=:all, all UTXOs will be returned.
      def list_unspent(only_finalized = true, label = :unlabeled)
        label = :unlabeled unless label
        wallet_adapter.list_unspent(id, only_finalized, label)
      end
```